c7n/filters/offhours.py

```diff
@@ -311,6 +311,13 @@ class Time(Filter):
         'utc': 'Etc/UTC',
     }
 
+    z_names = list(zoneinfo.get_zonefile_instance().zones)
+    non_title_case_zones = (
+        lambda aliases=TZ_ALIASES.keys(), z_names=z_names:
+        {z.lower(): z for z in z_names
+            if z.title() != z and z.lower() not in aliases})()
```

Comment:
Ignoring timezones that show up in `z_names` that are already in `TZ_ALIASES` (like `est`) since `zonefile('EST') != zonefile('US/Eastern')`, `EST` does not follow daylight savings, whereas `US/Eastern` or `US/New_York` does.